pandas/core/arrays/datetimes.py

```diff
@@ -1874,6 +1898,7 @@ def to_julian_date(self) -> npt.NDArray[np.float64]:
                 self.hour
                 + self.minute / 60
                 + self.second / 3600
+                + self.millisecond / 3600 / 10**3
```

Comment:
Why was this added? I think `self.microsecond` should already include the milliseconds